Library/Homebrew/official_taps.rb

```diff
@@ -8,6 +8,14 @@
   tex
 ].freeze
 
+OFFICIAL_CASK_TAPS = %w[
```

Comment:
There may also be a way to use a smarter GitHub search that involves hitting one endpoint for multiple taps rather than multiple endpoints.